docs/user-guide/layer-settings.md

```diff
@@ -321,6 +321,18 @@ It this case the editor is allowed to choose a `Color ramp` and the order (with
 
 <img src="../img/layer-settings/classification_style_ex.jpg" class="ms-docimage">
 
+!!!note
+    The *Classification style* method is also available for **WFS** and **Vector** layers.
+
+!!!note
+
+    From the *Classification style* options, it is possible to activate the **Thematic Layer** option thought the <img src="../img/button/configute_lite_button.jpg" class="ms-docbutton"/> button. 
+    <img src="../img/layer-settings/configure_popup.jpg" class="ms-docimage" style="max-width:400px;">
```

Comment:
Use this. Optimized it for the web, please.
![image](https://github.com/geosolutions-it/MapStore2/assets/1280027/05f4587e-ca03-4657-addb-537c19f65a6d)